cli/commands/provision/README-nicegui.md

```
@@ -0,0 +1,416 @@
+## Quick Start
+
+```bash
+# Start web server (auto-opens browser)
+uv run --with nicegui python nicegui_app.py
+
+# Opens http://localhost:8080
+```
+
+## Screenshot
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)